strictdoc/export/html/_static/controllers/autocompletable_field_controller.js

```diff
@@ -199,6 +231,7 @@
         this.element.dispatchEvent(new CustomEvent("loadstart"))
         const html = await this.doFetch(url)
         this.replaceResults(html)
+        this.selectText(this.autocompletable.innerText.trim());     /* select matching entry */
```

Comment:
Let's have the comment above the line. StrictDoc most of the time does not have the right-side comments, except when they are about mypy, ruff, coverage, etc.

Let's also have it start with a capital S and end with a dot. I am planning to extend the dev guide soon with more guides about the comments. See also my other comment about the traceable dev guide plan.